### PR TITLE
Added Dockerfile to run privacy_bot and a simple script to install an…

### PR DESCRIPTION
**Pull request by fedelodo**
I've added a simple dockerfile and a script to run the docker container.

### CODE REVIEW

**Comment by remusao** [comment]
Hey @fedelodo, thanks a lot for your work! Would you like to join us on gitter https://gitter.im/munich-cliqz/Lobby ? I will review your PR!

**Comment by remusao** [comment]
I just noticed that all your files are in a folder named: `privacy_bot_docker/`, any reason to not put them at the root of the repository?

File: privacy_bot_docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.5
+
+ADD privacy_bot /tmp/privacy_bot
```
Comment (remusao) [inline privacy_bot_docker/Dockerfile]: What about `ADD . /tmp/privacy_bot` to avoid having the code in a sub-folder?

**Comment by remusao** [comment]
Hi @fedelodo 
I tried the PR, and it almost works, I just had to change this in Dockerfile:
```
-COPY privacy-bot/ /tmp/privacy_bot
+COPY privacy_bot/ /tmp/privacy_bot
```

Would you mind changing it quick and then I can merge it :)

**Comment by fedelodo** [comment]
Sure! Sorry I've been a bit busy today!

**Pull request merged**